polaris/polaris/models.py

```diff
@@ -28,36 +27,14 @@
 from stellar_sdk.exceptions import SdkError
 
 
-# This dictionary acts as an in-memory indication of whether or not
-# the Asset object's distribution account fields have been updated
-# since this process started.
-ASSET_DISTRIBUTION_ACCOUNT_LOADED = {}
+# Used for loading the distribution signers data onto an Asset obj
+ASSET_DISTRIBUTION_ACCOUNT_DATA = {}
```

Comment:
nit: `ASSET_DISTRIBUTION_ACCOUNT_DATA` naming is a little confusing to me. It's a dictionary with keys being asset codes, and values being distribution account data?

If so, could we name it to something like `ASSET_TO_DISTRIBUTION_ACCOUNT_MAP` ?

Reply:
Thats a better name, thanks.